### PR TITLE
Updating quantization documentation

### PR DESCRIPTION
**Pull request by HDCharles**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #68907

Summary: Added information about symmetric
qschemes and corrected an error in reference to https://github.com/pytorch/pytorch/issues/68540

Test Plan: cd docs
make html
python -m http.server
// open webpage, inspect it, looks good
Screen Shot 2021-12-06 at 10 42 47 PM
(full mathematical description of quantization process)

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D32662033](https://our.internmc.facebook.com/intern/diff/D32662033)

### CODE REVIEW

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32662033).

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32662033).

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32662033).

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32662033).

**Comment by vkuzo** [comment]
thoughts about reverting the `ao` change for now?  The refactor to `torch.ao` is not user facing yet as we haven't finished the migration and havent published the OSS summary of what's happening.

File: docs/source/quantization-support.rst [hunk 1]
```diff
@@ -440,7 +440,10 @@ Note that operator implementations currently only
 support per channel quantization for weights of the **conv** and **linear**
 operators. Furthermore the minimum and the maximum of the input data is
 mapped linearly to the minimum and the maximum of the quantized data
-type such that zero is represented with no quantization error.
+type such that zero is represented with no quantization error. This is slightly altered
+for symmetric qschemes, which instead use the negative and positive of the maximum absolute
+value of the input data in place of the normal minimum and maximum for the input data. For more info see how
+scale and zero point are calculated by :class:`~torch.ao.quantization.observer.MinMaxObserver`.
```
Comment (vkuzo) [inline docs/source/quantization-support.rst:446]: this makes sense.  Thoughts about just writing out the formulas in addition to the text, to remove even more ambiguity?

Comment (HDCharles) [reply docs/source/quantization-support.rst:446]: ya, good idea

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32662033).

File: docs/source/quantization-support.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Quantization API Reference
 -------------------------------
 
-torch.quantization
+torch.ao.quantization
```
Comment (vkuzo) [inline docs/source/quantization-support.rst:4]: should we remove the `ao` for now, since the migration is not user facing yet?

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32662033).

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32662033).

File: docs/source/quantization-support.rst (hunk 2 above)
Comment (HDCharles) [reply docs/source/quantization-support.rst]: done

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32662033).

**Review by vkuzo — APPROVED** [review]
Thanks for doing this!

**Pull request closed without merging**